specs/windows/dns_cache.table

```diff
@@ -0,0 +1,11 @@
+table_name("dns_cache")
+description("DNS Cache.")
```

Comment:
Can you elaborate on this, for example, include that this uses the undocumented DnsGetCacheDataTable Windows API.